packages/function-entry/src/hints-popover.jsx

```diff
@@ -0,0 +1,142 @@
+import { withStyles } from 'material-ui/styles';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popover from 'material-ui/Popover';
+import Typography from 'material-ui/Typography';
+
+const styles = theme => ({
+  fraction: {
+    display: 'inline-block',
+    position: 'relative',
+    'vertical-align': 'middle',
+    'letter-spacing': '0.001em',
+    'text-align': 'center',
+    'margin-right': theme.spacing.unit,
+  },
+  'fraction-span': {
```

Comment:
use `fractionSpan` instead of `'fraction-span'`? then in the comp you can go: `<Comp className={classes.fractionSpan}  />` instead of `<Comp className={classes['fraction-span']}/>`?